src/frontend/sharedComponents/GPSPill.test.tsx

```diff
@@ -0,0 +1,45 @@
+import {render, screen} from '@testing-library/react-native';
+
+import {GPSPill} from './GPSPill';
+
+test.skip('searching status', async () => {
+  render(<GPSPill status="searching" />);
+
+  // TODO: Check icon color
+});
```

Comment:
there's no text that's rendered from what I can tell (instead it's the activity indicator). not really sure how to query that as of right now but open to adding an assertion if there's a clear way.